File: backend/src/main/kotlin/com/back/back9/domain/orders/repository/OrdersRepository.java
package com.back.back9.domain.orders.repository;

import com.back.back9.domain.orders.entity.Orders;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface OrdersRepository extends JpaRepository<Orders, Long> {
    List<Orders> findByWalletId(Long walletId);
}

File: backend/src/main/kotlin/com/back/back9/domain/orders/repository/OrdersRepository.kt
package com.back.back9.domain.orders.repository

import com.back.back9.domain.orders.entity.Orders
import org.springframework.data.jpa.repository.JpaRepository

interface OrdersRepository : JpaRepository<Orders?, Long?> {
    fun findByWalletId(walletId: Long?): MutableList<Orders>
}
